### ISSUE TITLE
[DEV] Porting "Observers" as a AC conf

### ISSUE THREAD

**Issue by Helias** [azerothcore/forum#121]
The patch "Observers" allows to "observ" the players each other or to spy player using a GM, this can be a good feature for AC, we just need to port it into AC as a conf or may as a module (but I don't think it's simple).

https://github.com/trickerer/Observers

**Comment by IceNet** [comment]
"How to stalk a player as a player"

**Comment by BarbzYHOOL** [comment]
that's really interesting.

What does @trickerer think about porting this to AC too?

Note: Deku had started a similar mod but only for GMs, I still know where his code is but it's not completed sadly

**Comment by trickerer** [comment]
So, I took a peek at AC current code. Observers patch is 90% compatible with it. I don't have time to compile & carefully test it, but I can prepare a patch for AC. You deal with minor compile errors if any.

**Comment by Helias** [comment]
I think that we can include this Observer patch as an official feature of AC may disabled by conf

**Comment by BarbzYHOOL** [comment]
yes, apply the patch, fix the issues and then make a real PR helias :P
